foundationdb/src/tuple/mod.rs

```diff
@@ -139,7 +163,6 @@ macro_rules! tuple_impls {
 }
 
 tuple_impls! {
-    1 => (0 T0)
```

Comment:
By removing this, I think it clarifies single tuple vs. single element in a lot of contexts.